sdk/lib/crt/startup/tlsmthread.c

```diff
@@ -20,11 +20,6 @@ extern int ___w64_mingwthr_add_key_dtor (DWORD key, void (*dtor)(void *));
 
```

Comment:
But all the rest in this file is still needed? o_O

Reply:
fix gcc.

Reply:
OK no problem, I just want also to be sure that this file is not used at all when compiling with MSVC.